src/test/java/no/unit/nva/model/BookPublicationTest.java

```diff
@@ -21,6 +30,19 @@ void publicationReturnPublicationWhenInputIsValid() throws MalformedContributorE
 
         Publication publicationFromJson = objectMapper.readValue(objectMapper.writeValueAsString(document),
                 Publication.class);
-        Assertions.assertEquals(bookPublication, publicationFromJson);
+
+        assertThat(bookPublication, is(equalTo(publicationFromJson)));
+    }
+
+    @Test
+    void publicationReturnsSeriesWhenInputIsSeriesTitle() throws MalformedContributorException, InvalidIsbnException,
```

Comment:
Create  a test describing the desired response when both a an unconfirmed series title and a confirmed series are present